components/scream/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -1,50 +1,59 @@
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/kernels/mo_optical_props_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/kernels/mo_rte_solver_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_fluxes_byband.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_fluxes_byband_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_fluxes_bygpoint.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_heating_rates.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_rrtmgp_clr_all_sky.F90
+  rrtmgp_inputs_initializer.cpp
+  tests/rrtmgp_test_utils.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rrtmgp/kernels/mo_gas_optics_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rrtmgp/mo_rrtmgp_constants.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rrtmgp/mo_rrtmgp_util_reorder.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/mo_rte_lw.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/mo_rte_sw.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/expand_and_transpose.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/kernels/mo_fluxes_broadband_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/kernels/mo_optical_props_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/kernels/mo_rte_solver_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/examples/mo_load_coefficients.cpp
+  # This is a hack; I should not have to build this here, but we need it for the standalone RRTMGP
+  # unit tests, and I could not figure out how to get it to build without adding it to the RRTMGP
+  # library. TODO: revisit this!
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/examples/all-sky/mo_garand_atmos_io.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/examples/all-sky/mo_load_cloud_coefficients.cpp
 )
 
 set(RRTMGP_HEADERS
   rrtmgp.hpp
   scream_rrtmgp_interface.hpp
   atmosphere_radiation.hpp
+  rrtmgp_inputs_initializer.hpp
 )
 
 add_library(rrtmgp ${RRTMGP_SRCS})
+set_source_files_properties(${RRTMGP_SRCS} PROPERTIES COMPILE_FLAGS "${YAKL_CXX_FLAGS}")
 target_include_directories(rrtmgp PUBLIC ${SCREAM_INCLUDE_DIRS})
-target_include_directories(rrtmgp PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../share)
-target_include_directories(rrtmgp SYSTEM PUBLIC ${SCREAM_TPL_INCLUDE_DIRS} ${CIMEROOT}/src/share/include)
-set_target_properties(rrtmgp PROPERTIES
-      Fortran_MODULE_DIRECTORY ${SCREAM_F90_MODULES})
-target_link_libraries(rrtmgp physics_share scream_share ${SCREAM_TPL_LIBRARIES})
+target_include_directories(rrtmgp PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../common)
+#target_include_directories(rrtmgp SYSTEM PUBLIC ${SCREAM_TPL_INCLUDE_DIRS} ${CIMEROOT}/src/share/include)
```

Comment:
We can get rid of stuff like this, unless you think we'll be adding it back in later.

Reply:
Thanks, done.